internal/generate/olm-catalog/csv_updaters.go

```diff
@@ -229,20 +231,64 @@ var _ csvUpdater = crds{}
 // apply will only make a new spec.customresourcedefinitions.owned element for
 // a type if an annotation is present on that type's declaration.
```

Comment:
@estroz I don't think this is true right?
I can have no CSV annotations defined on my types and the CSV generator will still generate the bare minimum `spec.customresourcedefintions.owned` with my gvks populated.

Reply:
Yes this comment should have been removed in a preceding PR.